data/json/recipes/recipe_food.json

```diff
@@ -4228,17 +4229,24 @@
 },{
   "type" : "recipe",
   "result": "syrup",
+  "id_suffix": "with_instrument",
```

Comment:
Is there someone currently working on that? It would be nice if optional tools could reduce the base time, the amount of some components and the difficulty.